src/NHSD.BuyingCatalogue.SolutionLists.API/ViewModels/ListSolutionsFilterViewModel.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using NHSD.BuyingCatalogue.SolutionLists.Contracts;
+
+namespace NHSD.BuyingCatalogue.SolutionLists.API.ViewModels
+{
+    /// <summary>
+    /// Provides the filter criteria for the <see cref="ListSolutionsQuery"/> query.
+    /// </summary>
+    public sealed class ListSolutionsFilterViewModel : IListSolutionsQueryData
+    {
+        public ISet<CapabilityReferenceViewModel> Capabilities { get; } = new HashSet<CapabilityReferenceViewModel>();
+
+        [JsonIgnore]
+        public ISet<ICapabilityReference> CapabilityReferences
+        {
+            get
+            {
+                return Capabilities.OfType<ICapabilityReference>().ToHashSet();
+            }
+        }
+
+        /// <summary>
+        /// Filters to only foundation solutions
+        /// </summary>
+        public bool IsFoundation { get; set; } = false;
```

Comment:
At the moment is looks like we have two endpoints to filter by foundation solutions seen as this is exposed & we have the /Foundation endpoint. Could raise another task to resolve this.

Reply:
👍 I'll create a new tech debt task to resolve this